pkg/blackduck/ctl_blackduck_test.go

```diff
@@ -257,6 +257,20 @@ func TestSetCRSpecFieldByFlag(t *testing.T) {
 			},
 		},
 		// case
+		{
+			flagName: "postgres-init-post-command",
+			changedCtl: &HelmValuesFromCobraFlags{
+				flagTree: FlagTree{
+					PostCommand: "",
```

Comment:
Can we put can actual value into here?

Reply:
done --> default is empty. customer will set only in case of Istio..